src/nnet3/nnet-normalize-component.cc

```diff
@@ -1228,28 +1231,68 @@ void BatchRenormComponent::Write(std::ostream &os, bool binary) const {
   moving_mean_.Write(os, binary);
   WriteToken(os, binary, "<MovingStddev>");
   moving_stddev_.Write(os, binary);
+  WriteToken(os, binary, "<AverageCount>");
+  WriteBasicType(os, binary, average_count_);
   WriteToken(os, binary, "</BatchRenormComponent>");
 }
 
-void BatchRenormComponent::Scale(BaseFloat scale) {
+void BatchRenormComponent::Scale_Training(BaseFloat scale) {
```

Comment:
This is against the Google style guide; should be ScaleTraining.
However, I think it would be better to just use the regular Scale() function, you can see how I've done it in my version.